prusti/tests/verify/todo/solve-ax-b-0.rs

```diff
@@ -0,0 +1,13 @@
+#![feature(rustc_private)]
+
+extern crate prusti_contracts;
+
+#[requires="a != 0f64"]
+#[ensures="a*result + b == 0f64"]
+fn solve(a: f64, b: f64) -> f64 {
+    -b/a
+}
+
+fn main() {
+    solve(5f64, 2f64);
+}
```

Comment:
Floating point numbers are currently not supported, but Prusti should not crash this bad.

Could you please remove this program from the PR, and move it to a new issue "Crash when verifying unsupported types"?